griptape/tasks/base_audio_generation_task.py

```diff
@@ -13,9 +13,9 @@ class BaseAudioGenerationTask(BlobArtifactFileOutputMixin, RuleMixin, BaseTask,
     def before_run(self) -> None:
         super().before_run()
 
-        self.structure.logger.info(f"{self.__class__.__name__} {self.id}\nInput: {self.input.to_text()}")
+        self.structure.logger.info("%s %s\nInput: %s", self.__class__.__name__, self.id, self.input.to_text())
```

Comment:
This is one case where I might favor readability over the micro-optimization of lazy string formatting, but 🤷

Reply:
Agreed, but 🤷